### PR TITLE
Required signature standards

### PR DESCRIPTION
**Pull request by CarterKoala**

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -446,14 +446,15 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
         - A7a1) The judge should indicate to the competitor that the puzzle is solved, or that a penalty will be applied.
-    - A7b) The judge records the result on a score sheet and signs next to the result to acknowledge that the result is correct, complete, properly formatted, and clearly readable.
+    - A7b) The judge records the result on a score sheet and signs next to the result to acknowledge that the result is correct, complete, properly formatted, and clearly readable. Judge signatures should be consistent throughout the competition and should identify the judge.
         - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65).
     - A7c) The competitor must check the recorded result, and sign (or initial) the score sheet to acknowledge that the result is correct, complete, properly formatted, and clearly readable. This finishes the attempt.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
    - A7c) The competitor must check the recorded result, and sign the score sheet to acknowledge that the result is correct, complete, properly formatted, and clearly readable. This finishes the attempt.
```
Initials can now be considered a signature due to A7c++

File: wca-regulations.md [hunk 2]
```diff
@@ -397,7 +397,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
-        - A2d1) The scrambler or an authorized judge must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image (see [Regulation 4g](regulations:regulation:4g)). Exception: for 6x6x6 Cube, 7x7x7 Cube, and Megaminx the scrambler may sign (or initial) to affirm that the puzzle is sufficiently scrambled (see [Regulation 4g1](regulations:regulation:4g1)).
+        - A2d1) The scrambler or an authorized judge must check the scrambled puzzle, and sign (or initial) the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image (see [Regulation 4g](regulations:regulation:4g)). Exception: for 6x6x6 Cube, 7x7x7 Cube, and Megaminx the scrambler may sign (or initial) to affirm that the puzzle is sufficiently scrambled (see [Regulation 4g1](regulations:regulation:4g1)). Scrambler signatures should be consistent throughout the competition and should identify the scrambler.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
        - A2d1) The scrambler or an authorized judge must check the scrambled puzzle, and sign the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image (see [Regulation 4g](regulations:regulation:4g)). Exception: for 6x6x6 Cube, 7x7x7 Cube, and Megaminx the scrambler may sign to affirm that the puzzle is sufficiently scrambled (see [Regulation 4g1](regulations:regulation:4g1)). Scrambler signatures should be consistent throughout the competition and should identify the scrambler.
```
Initials can now be considered a signature due to A7c++

**Comment by xsrvmy** [comment]
Not sure if this is still standard practice in China, but when I competed in China, I was asked to sign in full on the first attempt then initial the rest. This would go against the recommended practice of these new regs.

**Comment by ohexter** [comment]
> Not sure if this is still standard practice in China, but when I competed in China, I was asked to sign in full on the first attempt then initial the rest. This would go against the recommended practice of these new regs.

A7c6 is a "should" so Delegates can override this if they have a strong reason, but we generally believe consistency is desirable.

**Review by ohexter — APPROVED** [review]

**Pull request merged**